### ISSUE TITLE
About attacking the vgg16 model

### ISSUE THREAD

**Issue by tuji-sjp** [dongyp13/Translation-Invariant-Attacks#6]
Hello, I have a question about attacking the vgg16 model.
There is no "AuxLogits", "Predictions", "logits" ....  in VGG16, but these variables are necessary in your TI algorithm.
So what should I do? Please help me, thank you very much!

**Comment by dongyp13** [comment]
Yes, attacking VGG-16 is different: 1) VGG-16 needs 224*224 inputs, you need to resize the image before feeding to the classifier. 2) the number of output classes is 1000 instead of 1001. 3) VGG-16 has the "logits" outputs, and predicted label can be inferred from logits. 

I will provide the code for attacking VGG-16 soon.

**Comment by dongyp13** [comment]
I've update the code at `attack_vgg16_mim.py`.

**Comment by tuji-sjp** [comment]
Thank you very much!

**Comment by tuji-sjp** [comment]
Sorry to bother you again. I have another silly question.
Does the adversarial example need to be pre-processed (such as normalization) before being fed into the model for reclassification?

**Comment by dongyp13** [comment]
Yes, of course. You need to preprocess adversarial images just like natural images.
